src/UnitTests/Core/Serialization/ArgumentDeserializerTests.cs

```diff
@@ -39,19 +40,22 @@ public class ArgumentDeserializerTests
         private ProcedureSerializer sigser;
         private ArgumentDeserializer argser;
         private MsdosPlatform platform;
+        private MockRepository mr;
 
         [SetUp]
         public void Setup()
         {
+            mr = new MockRepository();
             var sc = new ServiceContainer();
             sc.AddService<IFileSystemService>(new FileSystemServiceImpl());
             arch = new X86ArchitectureReal();
             platform = new MsdosPlatform(sc, arch);
-            sigser = new X86ProcedureSerializer(
+            sigser = mr.Stub<ProcedureSerializer>(
                 arch,
                 new TypeLibraryDeserializer(platform, true, new TypeLibrary()),
                 "stdapi");
-            argser = new ArgumentDeserializer(sigser, arch, arch.CreateFrame(), 4);
+            argser = new ArgumentDeserializer(sigser, arch, arch.CreateFrame(), 13);
```

Comment:
Why I set it to 13? I think, it would be nice for unit tests to cover such strange cases